src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java

```diff
@@ -19,6 +19,14 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member targetMember = DiscordUtils.getMember(context.getMessage(0), context.getGuild());
+        if (targetMember.getUser() == sender.getUser()) {
+            context.replyWarning("Why would I kick you~");
```

Comment:
Possibly a better phrasing?
```suggestion
            context.replyWarning("Why would you want to kick yourself~");
```